pom.xml

```diff
@@ -313,6 +313,7 @@
 					<excludes>
 						<exclude>it.**</exclude>
 					</excludes>
+					<argLine>-Dfile.encoding=${project.build.sourceEncoding}</argLine>
```

Comment:
This effectively sets the default file encoding to UTF-8. This means you will not catch cases where the file encoding needs to be UTF-8, but is not properly set.
It might be bettter to set it to a charset that allows to detect that situation easily.